main.tf

```diff
@@ -4,4 +4,51 @@ This file is used to implement the ROOT module.
 
 For more info refer https://github.com/terraform-ibm-modules/getting-started
```

Comment:
Add output.tf file at root level

Reply:
added output.tf

Reply:
Did you add output.tf at root level ... unable to see that ... please check

Reply:
I mean output.tf file exists but its contents are empty

Reply:
I was using Ann's example and didn't recognize the issue when I working on it.  With CP4S we did not have an output for the result when the installation finished. When the installation finishes a user can not log into the cloud pak until they have configured an LDAP with the install.  We would like to add that addition in the future however at this time we have not automated LDAP and a proper output.tf can't be created because of this.